src/cudamatrix/cu-vector.cc

```diff
@@ -328,7 +328,7 @@ MatrixIndexT CuVectorBase<Real>::ApplyFloor(Real floor_val) {
     cuda_vec_apply_floor(dimGrid, dimBlock, data_, floor_val, count_vec.Data(), dim_);
     CU_SAFE_CALL(cudaGetLastError());
     num_floored = count_vec.Sum();
-    CuDevice::Instantiate().AccuProfile("CuVectorBase::ApplyFloor", tim.Elapsed());
+    CuDevice::Instantiate().AccuProfile(__func__, tim.Elapsed());
```

Comment:
Yiming, the reason we sometimes write e.g. CuVectorBase::ApplyFloor instead of __func__, is that when the same function exists in multiple classes (e.g. also in  CuMatrixBase), we otherwise wouldn't know which one was taking time.